packages/e2e-tests/plugins/custom-grouping-block/index.js

```diff
@@ -1,16 +1,19 @@
 ( function () {
-	wp.blocks.registerBlockType( 'test/alternative-group-block', {
+	const el = wp.element.createElement;
+	const { InnerBlocks, useBlockProps } = wp.blockEditor;
+	const { createBlock, registerBlockType } = wp.blocks;
+
+	registerBlockType( 'test/alternative-group-block', {
+		apiVersion: 3,
```

Comment:
I got here from this comment: https://github.com/WordPress/gutenberg/pull/69932#discussion_r2247839289

In the future, all editors will likely run in iframes, so I think this approach is fine. If the other tests failing in this PR are also due to non-iframe editors, we might be able to change them to test in the iframe editor.